relay_messages.go

```diff
@@ -272,3 +342,13 @@ func finishesCall(f *Frame) bool {
 		return false
 	}
 }
+
+// isCallResOK indicates whether the call was successful
+func isCallResOK(f *Frame) bool {
+	return f.Payload[_resCodeIndex] == _resCodeOK
+}
```

Comment:
should we use this method to implement `lazyCallRes.OK` (so we don't have 2 different ways to read this field)

also means we can drop a few fields from the `lazyCallRes` struct